examples/bigquery-analyze-realtime-reddit-data/terraform/main.tf

```diff
@@ -0,0 +1,284 @@
+# service account for reddit VM
+resource "google_service_account" "reddit_service_account" {
+  account_id   = "${var.service_account_name}"
+  display_name = "Reddit VM Service Account"
+}
+
+resource "google_project_iam_binding" "reddit_vm_compute" {
+  project = "${var.project_id}"
+  role    = "roles/compute.admin"
+  members = [
+    "serviceAccount:${google_service_account.reddit_service_account.email}"
+  ]
+}
+
+resource "google_project_iam_binding" "reddit_vm_dataflow" {
+  project = "${var.project_id}"
+  role    = "roles/dataflow.admin"
+  members = [
+    "serviceAccount:${google_service_account.reddit_service_account.email}"
+  ]
+}
+
+resource "google_project_iam_binding" "reddit_vm_pubsub" {
+  project = "${var.project_id}"
+  role    = "roles/pubsub.admin"
+  members = [
+    "serviceAccount:${google_service_account.reddit_service_account.email}"
+  ]
+}
+
+
+resource "google_compute_instance" "reddit_vm" {
+  name         = "reddit-data-collector-1"
+  machine_type = "e2-micro"
+  zone         = "us-central1-a"
+
+
+  boot_disk {
+    initialize_params {
+      image = "centos-cloud/centos-7"
+    }
+  }
+
+  network_interface {
+    network = "default"
+
+    access_config {
+      // Ephemeral public IP
+    }
+  }
+
+  metadata_startup_script = <<EOF
+    echo \"install python and git...\"
+    yum install python3 -y
+    yum install git -y
+    echo \"installing pip...\"
+    curl https://bootstrap.pypa.io/pip/3.6/get-pip.py -o get-pip.py
+    python3 get-pip.py
+    python3 -m pip install --upgrade pip
+    echo \"importing python libraries...\"
+    python3 -m pip install praw
+    python3 -m pip install pandas
+    python3 -m pip install textblob
+    python3 -m pip install better_profanity
+    python3 -m pip install google-cloud-dataflow-client
+    python3 -m pip install textstat
+    python3 -m textblob.download_corpora
+    python3 -m pip install google-cloud-pubsub
+    echo \"downloads complete...\"
+    echo \"creating dirs for git repo...\"
+    mkdir reddit
+    cd reddit
+    git init
+    git clone https://github.com/CYarros10/reddit-streaming-application.git
```

Comment:
Why not add the python source to this repo and put them under a proper license? The referenced repo has no OSS licensing information at all.

Reply:
@CYarros10 I think you now just have the change the coordinate here and you are good to go